tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -61,8 +61,7 @@ class InternalCallMapHandlerTest extends TestCase
         '/^redis/', // redis extension
         '/^imagick/', // imagick extension
         '/^uopz/', // uopz extension
-        '/^memcache_/', // memcache extension
-        '/^memcache::/', // memcache extension
+        '/^memcache[_:]/', // memcache extension
```

Comment:
This is unrelated but not worth a separate PR. Simplified pattern to reduce number of patterns applied to every callmap (we'll add more in the future).